Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
     <PropertyGroup>
         <!-- This should be passed from the VSTS build -->
-        <MicrosoftIdentityAbstractionsVersion Condition="'$(MicrosoftIdentityAbstractionsVersion)' == ''">9.1.0</MicrosoftIdentityAbstractionsVersion>
```

Comment:
Why not 9.2.0?

Reply:
Because Certificate is now `GetCertificate()` / `SetCertificate()`